beta/src/pages/learn/thinking-in-react.md

```diff
@@ -25,45 +26,47 @@ The JSON API returns some data that looks like this:
 ]
 ```
 
-The mockup looks like this:
+मॉकअप इस तरह दिखता है:
 
 <img src="/images/docs/s_thinking-in-react_ui.png" width="300" style={{margin: '0 auto'}} />
 
-To implement a UI in React, you will usually follow the same five steps.
+React  में  UI  को लागू करने के लिए, आप आमतौर पर एक ही पांच चरणों का पालन करेंगे।
+
+## Step 1: एक कुम्पोनेंट पदानुक्रम में UI को तोड़ो {/*step-1-break-the-ui-into-a-कौम्पोनॅन्ट-hierarchy*/}
 
-## Step 1: Break the UI into a component hierarchy {/*step-1-break-the-ui-into-a-component-hierarchy*/}
+मॉकअप में प्रत्येक कॉम्पोनेन्ट और सब कंपोनेंट के चारों ओर बॉक्स बनाकर शुरू करें और उनका नामकरण करें।
+यदि आप किसी डिज़ाइनर के साथ काम करते हैं, तो हो सकता है कि उन्होंने अपने डिज़ाइन टूल में इन कॉम्पोनेंट्स का नाम पहले ही रख लिया हो। उनके साथ चेक इन करें!
 
-Start by drawing boxes around every component and subcomponent in the mockup and naming them. If you work with a designer, they may have already named these components in their design tool. Check in with them!
+आपकी पृष्ठभूमि के आधार पर, आप किसी डिज़ाइन को विभिन्न तरीकों से घटकों में विभाजित करने के बारे में सोच सकते हैं:
```

Comment:
`घटकों` => `कॉम्पोनेन्टस`

Reply:
@kumard3 I did a typo here. actually it should be `कौम्पोनॅन्टस` 
can you update this.